### ISSUE TITLE
Issue with Progress Bar % Total

### ISSUE THREAD

**Issue by kerns** [verbb/formie#540]
Tabbing between pages creates this error in how % of progress total is calculated. 

https://user-images.githubusercontent.com/20254/128604922-2525f7b4-c37d-4d9b-aec6-6175f473bae3.mov

**Steps to reproduce**
1. Create a two page form
2. Enable page tabs and progress bars
3. Profit.

### Form settings

- Multi-page form: Yes
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: Latest as of August 8 2021
- Craft version: 3.7.X
- Multi-site: No

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.12",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.13](https://github.com/verbb/formie/releases/tag/1.4.13)
